Automattic/wp-calypso
# Framework: Remove keyboard shortcuts

#### Changes proposed in this Pull Request

Calypso's keyboard shortcuts feature is an artifact of its pre-OSS era, started in February 2015 and developed in the following months. It was never finished, though, and was never even put for broader testing. It has never been enabled in staging/production and is currently available only on dev and wpcalypso environments. 

My brief research displayed that nobody was using keyboard shortcuts, and there were folks who ended up inadvertently doing something with a shortcut. So it's time to clean that feature up. If we ever decide to have a keyboard shortcuts feature, we'd likely use the Gutenberg [Keyboard Shortcuts](https://github.com/WordPress/gutenberg/tree/trunk/packages/keyboard-shortcuts) package for a base.

This PR cleans all keyboard shortcuts functionality altogether, as well as some related reader state that was used only for keyboard shortcuts.

#### Testing instructions

* Verify Calypso builds with no errors.
* Smoke test Reader and inline help.
* Verify the removed code is not in use.
* Verify tests pass.
